<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jupitermoon</class>
 <widget class="QDialog" name="jupitermoon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jupiter events</string>
  </property>
  <widget class="astroDateTime" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="WeekdayLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Monday</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NowButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Now(UTC)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NextButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>30</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MinButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MaxButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="CMII">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CMII:</string>
   </property>
  </widget>
  <widget class="QLabel" name="alt_jup">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alt jup:</string>
   </property>
  </widget>
  <widget class="QLabel" name="JDLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>JD:</string>
   </property>
  </widget>
  <widget class="QLabel" name="alt_sun">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alt sun:</string>
   </property>
  </widget>
  <widget class="QLabel" name="EventLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Event: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="PreviousButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>30</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>astroDateTime</class>
   <extends>QDateTimeEdit</extends>
   <header>astroDateTime.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>MaxButton</sender>
   <signal>clicked()</signal>
   <receiver>jupitermoon</receiver>
   <slot>maxScale()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MinButton</sender>
   <signal>clicked()</signal>
   <receiver>jupitermoon</receiver>
   <slot>minScale()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NowButton</sender>
   <signal>clicked()</signal>
   <receiver>jupitermoon</receiver>
   <slot>resetDate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateTimeEdit</sender>
   <signal>dateTimeChanged(QDateTime)</signal>
   <receiver>jupitermoon</receiver>
   <slot>updateDateTime()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>maxScale()</slot>
  <slot>minScale()</slot>
  <slot>updateDateTime()</slot>
  <slot>resetDate()</slot>
  <slot>updateStep()</slot>
  <slot>calculate_next()</slot>
 </slots>
</ui>
